### PR TITLE
utility to generate txs

### PR DESCRIPTION
**Pull request by niktrix**
## Problem

Asana Link: https://www.notion.so/herdius/Go-tool-package-for-testing-d254db86bd144077a9c728385401430b

### CODE REVIEW

**Comment by lmahanand** [comment]
> @niktrix Can you also add a command to create secret key that will also generate herdius address?

You can refer this test case to get the secret key and create address: https://github.com/herdius/herdius-core/blob/master/crypto/secp256k1/secpk256k1_test.go#L49

You can import the libraries since they are in public git repository.

File: cmd/test/cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+func initConfig() {
+	if cfgFile != "" {
+		// Use config file from the flag.
+		viper.SetConfigFile(cfgFile)
+	} else {
+		// Find home directory.
+		home, err := homedir.Dir()
+		if err != nil {
+			//er(err)
```
Comment (cuonglm) [inline cmd/test/cmd/root.go]: panic?

File: cmd/test/cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		}
+		viper.AddConfigPath(home)
+		viper.SetConfigName(".")
+	}
+
+	viper.AutomaticEnv()
+
+	if err := viper.ReadInConfig(); err == nil {
```
Comment (cuonglm) [inline cmd/test/cmd/root.go]: I think it's better to log the error and returns, so we can know what's the problem.

File: cmd/test/testutils/util.go [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+	if err != nil {
+		log.Println("Error http post to endpoint :", endpoint)
+		return
+	}
+
+	defer response.Body.Close()
+	body, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		log.Println("Error reading ioutil")
+
```
Comment (cuonglm) [inline cmd/test/testutils/util.go]: Remove this blank line.

File: cmd/test/testutils/util.go [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+	getURL := cmd.endpoint + "/account/" + address
+	response, err := http.Get(getURL)
+	if err != nil {
+		log.Println("Failed to get response: ", err)
+		return
+	}
+	defer response.Body.Close()
+	body, readErr := ioutil.ReadAll(response.Body)
+	if readErr != nil {
+		log.Println(fmt.Sprintf("Failed to read http response %s.", readErr))
```
Comment (cuonglm) [inline cmd/test/testutils/util.go]: missing return?

**Pull request closed without merging**